packages/logger/README.md

```diff
@@ -51,7 +52,7 @@ log.info('hello, world');
 
 ## API
 
-### `new Logger({ [executions], [granules], [parentArn], [sender], [stackName], [version] })`
+### `new Logger({ [executions], [granules], [parentArn], [sender], [stackName], [version], [asyncOperationId] })`
```

Comment:
I was trying to keep these alphabetized for marc.

Reply:
oh my gosh. sorry didn't notice that. I will fix it!